cmd/cluster-agent/api/v1/languagedetection/language_detection.go

```diff
@@ -45,13 +45,14 @@ func InstallLanguageDetectionEndpoints(r *mux.Router, wmeta workloadmeta.Compone
 }
 
 var ownersLanguages OwnersLanguages
-var languageTTL time.Duration
 var ownersLanguagesOnce sync.Once
 
+var languageTTL time.Duration
```

Comment:
IMO it's easier to have a struct that we initialize with the right values than to rely on global states.

Reply:
I fully agree, thanks for the suggestion.

I updated the code.